src/v2/Apps/FairOrginizer/fairOrganizerRoutes.tsx

```diff
@@ -47,6 +57,13 @@ export const fairOrganizerRoutes: AppRouteConfig[] = [
           profile {
             __typename
           }
+          runningFairs: fairsConnection(first: 1, status: RUNNING) {
```

Comment:
**suggestion**: This might not ever happen, but, if an organizer happens to have two concurrently running fairs, it might be nice to have an explicit sort defined here.

```suggestion
          runningFairs: fairsConnection(first: 1, status: RUNNING, sort: START_AT_DESC) {
```

Reply:
@dblandin that's a good point
But maybe in case of several concurrently running fairs it'd be better to display all of them? Otherwise they won't be displayed anywhere on the page